libs/feature/search/src/lib/favorites/favorite-star/favorite-star.component.spec.ts

```diff
@@ -115,54 +115,90 @@ describe('FavoriteStarComponent', () => {
       describe('if record is not part of favorite', () => {
         beforeEach(() => {
           ;(favoritesService as any).myFavoritesUuid$.next([
-            'aaa',
-            'bbb',
-            'ccc',
+            component.record.uuid,
           ])
-          starToggle.newValue.emit(true)
           fixture.detectChanges()
         })
-        it('adds record to favorites', () => {
-          expect(favoritesService.addToFavorites).toHaveBeenCalledWith([
-            component.record.uuid,
-          ])
-          expect(favoritesService.removeFromFavorites).not.toHaveBeenCalled()
+        describe('if change is made from this component', () => {
+          beforeEach(() => {
+            starToggle.newValue.emit(true)
+            fixture.detectChanges()
+          })
+          it('adds record to favorites', () => {
```

Comment:
This test is a consequence of the click

Reply:
Yes, for when the record is added to favorites from the component.